mungegithub/github/github.go

```diff
@@ -72,6 +72,15 @@ var (
 	combinedStatusLifetime = 5 * time.Second
 )
 
+func suggestOauthScopes(resp *github.Response, err error) error {
```

Comment:
I guess I would just be frustrated as a user if this told me I was getting a 403 from a scope when the issue was from my group membership etc